viewing_party/main.py

```diff
@@ -0,0 +1,129 @@
+def create_movie(title, genre, rating):
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return None
+
+
+def add_to_watched(user_data, movie):
+    user_data = {
+        "watched": [movie]
+    }
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data = {
+        "watchlist": [movie]
+    }
+    return user_data 
```

Comment:
Similar to the above

```suggestion
def add_to_watchlist(user_data, movie):
    user_data["watchlist"].append(movie)
    return user_data 
```